scrapper.py

```diff
@@ -21,43 +40,160 @@ class IncorrectNumberOfArticlesError(Exception):
     """
 
 
+def _clean_text(text):
+    return re.sub(r"[\n\t ]+", " ", text).strip()
+
+
+def _get_page(link):
+    print("attempting to connect to %s", link)
+    #time.sleep(random.uniform(2.0, 4.0))
+    headers = Headers().generate()
+    print(headers)
+    response = requests.get(link,
+                            headers=headers,
+                            timeout=5)
+    if not response.ok:
+        return None
+    return BeautifulSoup(response.text, "html.parser")
+    # except requests.exceptions.ConnectionError:
```

Comment:
all unused code should be removed by the final check-in

Reply:
Done